modules/gapi/src/backends/onnx/gonnxbackend.cpp

```diff
@@ -143,6 +143,41 @@ class ONNXCompiled {
     void run();
 };
 
+static void appendExecutionProvider(Ort::SessionOptions          *session_options,
+                                    const cv::gapi::onnx::ep::EP &execution_provider) {
+    namespace ep = cv::gapi::onnx::ep;
+    switch (execution_provider.index()) {
+        case ep::EP::index_of<ep::OpenVINO>(): {
+             GAPI_LOG_INFO(NULL, "OpenVINO Execution Provider is selected.");
```

Comment:
Maybe it makes sense to display for which network (if it is not mentioned in the log earlier).

Reply:
As an option, yes